frontend/lib/norent/components/helmet.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from "react";
+import { Helmet } from "react-helmet-async";
+import { getImageSrc } from "../homepage";
+import { useSiteName } from "../../ui/page";
+import { AppContext } from "../../app-context";
+
+const favicon16 = getImageSrc("favicon-16x16", "png");
+const favicon32 = getImageSrc("favicon-32x32", "png");
+const favicon96 = getImageSrc("favicon-96x96", "png");
+
+// The public-facing FB App ID associated with the JustFixNYC Facebook organization
+const FB_APP_ID = "247990609143668";
```

Comment:
Can you file an issue on the NoRent milestone to consider converting this to a justfix environment variable?

Reply:
👍